litmus-portal/frontend/src/theme/index.tsx

```diff
@@ -93,6 +99,9 @@ function customTheme(options: ThemeOptions) {
         },
         gray: '#5D6173',
       },
+      menuOption: {
+        active: 'rgba(16, 155, 103, 0.1)',
```

Comment:
this is a color...can you put this in custom colors.
The entire menuOption object shift inside customColors